src/publish.proj

```diff
@@ -44,24 +41,36 @@
     <PushToBlobFeed ExpectedFeedUrl="$(ExpectedFeedUrl)"
```

Comment:
I don't know if this is going to actually work with the publish leg.

Reply:
Testing on a cloned build. https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_build/index?definitionId=7682&_a=completed

Reply:
https://github.com/dotnet/coreclr/blob/master/buildpipeline/DotNet-Trusted-Publish.json#L93

I think that expect a flat container structure.

Reply:
CoreFx uses UploadToAzure https://github.com/dotnet/corefx/blob/master/src/publish.proj#L26 for this.

Reply:
I thought symbols were supported on Sleet? It's only the test binaries that are flat.

Reply:
Sleet supports symbols publishing, so the only thing that would change is the source of the files from "packages" to "symbols". FeedUrl should be the same

Reply:
Oddly hitting, 'Could not load file or assembly 'SleetLib, Version=2.2.9.0, Culture=neutral, PublicKeyToken=3499a980e2629b6b'. The system cannot find the file specified.'

Reply:
sleet does support symbols that isn't the issue here. The issue is that you are conflating 2 different publishes. 

What is here works great for the final publish but it doesn't work for the individual leg publish which gathers things in a flat container so that the final publish can do the real publish to myget/blob feed. I think you need to mimic the publish in corefx for the individual legs and use UploadToAzure.

Reply:
So currently the intermediate step publishes the packages and the symbols to an intermediate feed, which is restored from in the test build. I'll see if I can preserve this behavior but if it ends up being blocking, will move back to the old style of UploadToAzure.

Reply:
The current PublishToBlob doesn't support this. Just switch back to UploadToAzure for those steps.